src/status_im/chat/styles/input/input.cljs

```diff
@@ -82,8 +82,8 @@
    :ios                 {:line-height min-input-height
                          :left        (+ 10 left)}})
 
-(defnstyle seq-input-text [left]
-  {:min-width           200
+(defnstyle seq-input-text [left container-width]
```

Comment:
Right single-line is perfect. 
I was more thinking something like `(max 200  (- container-width left))`Right single-like is perfect.